.env.example

```diff
@@ -6,6 +6,5 @@ PORT=<port number>
 
 REACT_APP_ENVIRONMENT=<STAGING | PRODUCTION>
 
-
-
-
+REACT_APP_PUBLIC_STRIPE_KEY_TEST=""
```

Comment:
we also need to add this in `amplify.yml`
like
```
- REACT_APP_SENTRY_DSN=${REACT_APP_SENTRY_DSN}
```

I'll add this envs to aws amplify